src/hooks/use-canvas.ts

```
@@ -0,0 +1,581 @@
+        if(!isDrawingRef.current)
+            return
+
+        isDrawingRef.current = false
+
+        if(freehandRafRef.current) {
+            window.cancelAnimationFrame(freehandRafRef.current)
+            freehandRafRef.current = null
+        }
+
+        const draft = draftShapeRef.current
+        const draft = draftShapeRef.current
```

Biome (2.1.2)

[error] Shouldn't redeclare 'draft'. Consider to delete it or rename it.

'draft' is defined here:


(lint/suspicious/noRedeclare)